crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -390,7 +390,9 @@ impl<'w, 's> Commands<'w, 's> {
     /// - [`spawn_batch`](Self::spawn_batch) to spawn entities with a bundle each.
     #[track_caller]
     pub fn spawn<T: Bundle>(&mut self, bundle: T) -> EntityCommands {
-        self.spawn_empty().insert(bundle)
+        let mut entity = self.spawn_empty();
+        entity.insert(bundle);
+        entity
```

Comment:
IMO the only clear win for consuming `self`, but that's because the spawn methods return `EntityCommands` by value either way